library.go

```diff
@@ -59,3 +59,57 @@ func (c *Client) modifyLibraryTracks(add bool, ids ...ID) error {
 	}
 	return nil
 }
+
+// UserHasAlbums checks if one or more albums are saved to the current user's
+// "Your Music" library.
+func (c *Client) UserHasAlbums(ids ...ID) ([]bool, error) {
+	if l := len(ids); l == 0 || l > 50 {
+		return nil, errors.New("spotify: UserHasAlbums supports 1 to 50 IDs per call")
+	}
+	spotifyURL := fmt.Sprintf("%sme/albums/contains?ids=%s", c.baseURL, strings.Join(toStringSlice(ids), ","))
+
+	var result []bool
```

Comment:
In many cases, the final `err` check can be omitted.  For example:
```
var result []bool
err := c.get(spotifyURL, &result)
return result, err
```

Reply:
You're right! I've made some changes thruout the codebase.